### PR TITLE
Makes braziers and torches craftable by Yautja

### PR DESCRIPTION
**Pull request by carlarctg**
# About the pull request

Braziers and torches can now be crafted with runed sandstone sheets by Yautja. Braziers need to be filled with wood and then lit. Torches just need to be lit.

Plasma casters and pistols are now heat sources so they can ignite braziers and torches.

# Explain why it's good for the game

More cool lodge detail

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
add: Braziers and torches can now be crafted with runed sandstone sheets by Yautja. Braziers need to be filled with wood and then lit. Torches just need to be lit.
add: Plasma casters and pistols are now heat sources so they can ignite braziers and torches.
/:cl:

### CODE REVIEW

**Review by Agameofscones — APPROVED** [review]
Sprites are aight

File: code/game/objects/structures/props.dm [hunk 1]
```diff
@@ -635,16 +635,70 @@
...
+	if(luminosity)
+		SetLuminosity(luminosity)
+
+/obj/structure/prop/brazier/frame
+	name = "empty brazier"
+	desc = "An empty brazier."
+	icon_state = "brazier_frame"
+	luminosity = 0
+
+/obj/structure/prop/brazier/frame/attackby(obj/item/W, mob/user)
```
Comment (Zonespace27) [inline code/game/objects/structures/props.dm:652]: one letter vars bad

File: code/game/objects/structures/props.dm [hunk 2]
```diff
@@ -635,16 +635,70 @@
...
+	if(!istype(W, /obj/item/stack/sheet/wood))
+		return ..()
+	var/obj/item/stack/wooden_boards = W
+	if(wooden_boards.amount < 5)
+		to_chat(user, SPAN_WARNING("Not enough wood!"))
+		return
+	wooden_boards.use(10)
```
Comment (Zonespace27) [inline code/game/objects/structures/props.dm:659]: requires 5, uses 10

File: code/game/objects/structures/props.dm [hunk 3]
```diff
@@ -635,16 +635,70 @@
...
+	new /obj/structure/prop/brazier/frame_woodened(src)
+	qdel(src)
+
+/obj/structure/prop/brazier/frame_woodened
+	name = "empty full brazier"
+	desc = "An empty brazier. Yet it's also full. What???"
+	icon_state = "brazier_frame_filled"
+	luminosity = 0
+
+/obj/structure/prop/brazier/frame_woodened/attackby(obj/item/W, mob/user)
```
Comment (Zonespace27) [inline code/game/objects/structures/props.dm:670]: one letter vars bad

File: code/game/objects/structures/props.dm [hunk 4]
```diff
@@ -635,16 +635,70 @@
...
 	density = FALSE
+	luminosity = 5
+
+/obj/structure/prop/brazier/torch/frame
+	name = "unlit torch"
+	desc = "It's a torch, but it's not lit."
+	icon_state = "torch_frame"
+	luminosity = 0
+
+/obj/structure/prop/brazier/torch/frame/attackby(obj/item/W, mob/user)
```
Comment (Zonespace27) [inline code/game/objects/structures/props.dm:690]: one letter vars bad

File: code/game/turfs/walls/walls.dm [hunk 5]
```diff
@@ -383,6 +383,11 @@
 		place_poster(W,user)
 		return
 
+	if(istype(W, /obj/item/prop/torch_frame))
```
Comment (Zonespace27) [inline code/game/turfs/walls/walls.dm:386]: feels unnecessary, could be a desc/examine

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/props.dm (hunk 2 above)
Comment (carlarctg) [reply code/game/objects/structures/props.dm:659]: erm

**Pull request merged**